backend/src/main/java/com/example/cms/CmsApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableAsync;
 
 @SpringBootApplication
+@EnableAsync
```

Comment:
To sprawia, że funkcje oznaczone za pomocą `@Async` są wykonywane na innym wątku przez co nie trzeba czekać aż funkcja się zakończy, żeby zwrócić odpowiedź z API

Reply:
Może się przydać przy wysyłaniu maili @BartoszDolata